repose-aggregator/components/filters/client-auth/src/main/java/org/openrepose/filters/clientauth/openstack/OpenStackAuthenticationHeaderManager.java

```diff
@@ -189,6 +189,7 @@ private void setTenant() {
             }
         } else if (!sendAllTenantIds && sendTenantIdQuality) {
             if (!this.isDelagable && !this.isTenanted) {
+                //TODO: Should this, or any other, instance of putHeader be changed to appendHeader?
```

Comment:
Thinking about it, I might add it to the agenda for a dev meeting. That way we can get input from the entire team. I'd be more comfortable with that then a spike or something where one person makes the decision.